certified-connectors/GMO Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,907 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "x-ms-deployment-version": "1.0.0",
+    "version": "1.0",
+    "title": "GMO Sign",
+    "description": "GMO Sign Connector helps organizations connect and automate how they prepare, sign, act on, and manage agreements.",
+    "x-ms-api-annotation": {
+      "status": "Production"
+    },
+    "contact": {
+      "name": "GMO Global Sign",
+      "url": "https://www.gmosign.com/"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://secure.gmosign.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.gmogshd.com/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Content and Files;Productivity"
+    }
+  ],
+  "host": "y4omo61inf.execute-api.ap-northeast-1.amazonaws.com",
+  "basePath": "/prd",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "basicAuth": {
+      "type": "basic"
+    }
+  },
+  "security": [
+    {
+      "basicAuth": []
+    }
+  ],
+  "paths": {
+    "/customer/connect": {
+      "x-ms-notification-content": {
+        "description": "Details for Webhook",
+        "schema": {
+          "$ref": "#/definitions/WebhookEnvelopeResponse"
+        }
+      },
+      "post": {
+        "summary": "When an envelope status changes (Connect)",
+        "description": "Triggers a new flow when an envelope status changes.",
+        "operationId": "CreateHookEnvelope",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "required": [
+                "name",
+                "urlToPublishTo",
+                "envelopeEvents"
+              ],
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "x-ms-summary": "接続名",
+                  "x-ms-visibility": "important"
+                },
+                "urlToPublishTo": {
+                  "type": "string",
+                  "x-ms-summary": "Webhook用URL",
+                  "x-ms-visibility": "internal",
+                  "x-ms-notification-url": true
+                },
+                "envelopeEvents": {
+                  "type": "string",
+                  "x-ms-summary": "トリガーを引く封筒状態",

```

Comment:
@GMO-GSHD-APIGW please use English strings for all connector files. You can refer to [English language strings guidelines](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1) and to [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance).